spark-statsd-reporter/src/main/java/org/apache/spark/metrics/sink/statsd/MetricType.java

```diff
@@ -1,11 +1,11 @@
 package org.apache.spark.metrics.sink.statsd;
 
 /**
+ * A subset of StatsD metric types used for Spark
  * @see <a href="https://github.com/etsy/statsd/blob/master/docs/metric_types.md">
  *        StatsD metric types</a>
  */
 interface MetricType {
-    String COUNTER = "c";
```

Comment:
There isn't any use for `COUNTERS` at all?

eg something like `totalJobsCompleted` for a dispatcher?

Reply:
There's a difference between codahale/dropwizard counter metric type and StatsD counter. Codahale counters used a lot across the codebase and mutated programmatically. However, when you try to report a counter type to StatsD, it treats it as an increment only and always increases target metric value by adding the value of codahale/dropwizard counter.

So in order to report counter values, they need to become gauges to avoid StatsD increments. This [comment](https://github.com/mesosphere/spark-build/blob/DCOS-54419-switch-statsd-to-gauges-and-tests-refactoring/spark-statsd-reporter/src/main/java/org/apache/spark/metrics/sink/statsd/StatsdReporter.java#L108-L109) contains similar explanation and an example

Reply:
> However, when you try to report a counter type to StatsD, it treats it as an increment only and always increases target metric value by adding the value of 

This makes sense to me. Counters by nature increment. Gauges report values that can increase/decrease. My question was more along the lines of "do we never report counter types to StatsD"? Or maybe codahale counter metrics are being mutated incorrectly when they should be treated/renamed as gauges.

Anyway, it's not an important point to debate; just a question on removing `COUNTER` as a type.